pycbc/psd/analytical_space.py

```diff
@@ -29,7 +29,7 @@
 """
 
 import numpy as np
-from astropy import constants
+from scipy.constants import c
```

Comment:
In general we should preference astropy for constants. Is there a reason to prefer the scipy version?

Reply:
When using `astropy.constants.c`, there will be a non-critical warning said "Cannot find reference 'c' in '__init__.py'", and also it needs to be converted to float type by additionally saying `c.value`. It's OK to change back, I'll do it later!